multi-node/aws/build

```diff
@@ -14,13 +14,17 @@ if [ -n "$(git diff --shortstat 2> /dev/null | tail -n1)" ]; then
     VERSION="${VERSION}+dirty"
 fi
 
-echo "Building kube-aws ${VERSION}..."
-rm -rf ./gopath
-link_path="${PWD}/gopath/src/github.com/coreos"
-mkdir -p $link_path
-ln -s ${PWD}/../../ "${link_path}/coreos-kubernetes"
+export GOPATH="$(pwd)/_output"
+export KUBEAWS="$GOPATH/src/github.com/coreos/coreos-kubernetes/multi-node/aws"
 
-GOPATH=${PWD}/Godeps/_workspace:${PWD}/gopath CGO_ENABLED=0 go build -ldflags "-X main.VERSION ${VERSION}" -a -tags netgo -installsuffix netgo -o bin/kube-aws github.com/coreos/coreos-kubernetes/multi-node/aws/cmd/kube-aws
-rm -r ./gopath
+if [ -d $GOAPTH ]; then
+    rm -rf $GOPATH
+fi
+
+mkdir -p $GOPATH/src
+cp -r ./vendor/* $GOPATH/src
+mkdir -p $KUBEAWS
+cp -r ./pkg $KUBEAWS/pkg
+cp -r ./cmd $KUBEAWS/cmd
```

Comment:
nit: this build dir could be lighter weight using symlinks rather than copying everything around